docs/administering/for-work.md

```diff
@@ -0,0 +1,112 @@
+[Sandstorm for Work](https://sandstorm.io/business) is the name for a collection of features that
+are typically used by companies. Customers can buy a _feature key_ to enable these features on a
+Sandstorm server running at their own organization. Special pricing, sometimes including free
+feature keys, are available to [non-profits and community groups](#special-pricing). For pricing,
+please read the information on the [main website](https://sandstorm.io/business). As an
+implementation detail, Sandstorm is fully open-source software.
+
+For now, feature keys are typically 60-day trial keys. This gives you time to evaluate Sandstorm for
+Work, and it gives us time to finish testing the payments system.
+
+## Sandstorm for Work features
+
+### Authentication providers: LDAP and SAML
+
+LDAP is a protocol for storing information of nearly any kind; it is typically used to store data
+about people who work at a company, including login credentials.  Sandstorm's LDAP support allows
+you to log into Sandstorm with a username and password that is checked against an LDAP store. We
+expect Sandstorm's LDAP support to be compatible with Microsoft Active Directory, OpenLDAP, and many
+other systems.
+
+SAML is a protocol for exchanging information about access control. SAML support is coming soon. If
+you need this feature, please feel free to [request a feature key
+today][https://sandstorm.io/business]. SAML support should be compatible with Shibboleth and other
+systems.
+
+To enable either of these login providers, take the following steps:
+
+- Log into your Sandstorm server.
+
+- Make sure you have enabled [Sandstorm for Work on your server](#enabling-sandstorm-for-work).
+
+- Click **Admin Settings** within your Sandstorm server; this should take you to `/admin/settings`.
+
+You should now see a checkbox that allows you to enable LDAP login.
+
+### Features coming soon
+
+We're still working on the following features:
+
+**Organization management.** Choose an LDAP group that, upon login, should automatically be given
+_user_ status on this Sandstorm server.
+
+**Group Management.** This will allow you to share a grain with everyone in an LDAP/Active Directory
```

Comment:
I'll adjust accordingly.

FWIW I did realize that, I just thought it'd be clearer to use an LDAP example, but it's now clear to me that limiting the example to LDAP makes clarity worse.